packages/app-project/src/components/ProjectHeader/ProjectHeader.spec.js

```diff
@@ -1,62 +1,151 @@
+
     before(function () {
-      wrapper = shallow(<ProjectHeader inBeta title={TITLE} />)
+      render(<Stories.InBeta />)
+      approvedIcon = screen.queryByLabelText('ProjectHeader.ApprovedIcon.title')
+      underReviewLabel = screen.queryByText('ProjectHeader.UnderReviewLabel.underReview')
+    })
+
+    it('should show the Under Review label', function () {
+      expect(underReviewLabel).to.exist() 
     })
 
-    it('should not render a `<UnderReviewLabel />` component', function () {
-      expect(wrapper.find(UnderReviewLabel)).to.have.lengthOf(1)
+    it('should not show the approved badge', function () {
+      expect(approvedIcon).to.be.null()
     })
   })
 
-  describe('when the project has available translations', function () {
+  describe('launch approved', function () {
+    let approvedIcon, underReviewLabel
+
     before(function () {
-      wrapper = shallow(<ProjectHeader availableLocales={['en', 'fr']} title={TITLE} />)
+      render(<Stories.LaunchApproved />)
+      approvedIcon = screen.queryByLabelText('ProjectHeader.ApprovedIcon.title')
+      underReviewLabel = screen.queryByText('ProjectHeader.UnderReviewLabel.underReview')
+    })
+
+    it('should not show the Under Review label', function () {
+      expect(underReviewLabel).to.be.null()
+    })
+
+    it('should show the approved badge', function () {
+      expect(approvedIcon).to.exist()
+    })
+  })
+
+  describe('in admin mode', function () {
+    it('should show the admin page link', function () {
+      render(<Stories.AdminMode />)
+      const navMenu = screen.getByRole('navigation', { name: 'ProjectHeader.ProjectNav.ariaLabel' })
+      const navLinks = within(navMenu).getAllByRole('link')
+
+      expect(navLinks.length).to.be.above(0)
+      expect(navLinks[0].href).to.equal('https://localhost/zooniverse/snapshot-serengeti/about/research')
+      expect(navLinks[navLinks.length - 1].href).to.equal('https://localhost/admin/project_status/zooniverse/snapshot-serengeti')
+    })
+  })
+
+  describe('with a default workflow', function () {
+    it('should display a workflow-specific classify link', function () {
+      render(<Stories.DefaultWorkflow />)
+      const navMenu = screen.getByRole('navigation', { name: 'ProjectHeader.ProjectNav.ariaLabel' })
+      const navLinks = within(navMenu).getAllByRole('link')
+
+      expect(navLinks.length).to.be.above(0)
+      expect(navLinks[0].href).to.equal('https://localhost/zooniverse/snapshot-serengeti/about/research')
+      expect(navLinks[1].href).to.equal('https://localhost/zooniverse/snapshot-serengeti/classify/workflow/1234')
+    })
+  })
+
+  describe('with multiple languages', function () {
+    let languageMenuItems, user
+    const projectLocales = ['en', 'fr', 'es']
+
+    before(async function () {
+      user = userEvent.setup({ delay: 'none' })
+      render(<Stories.MultipleLanguages />)
+      const languageButton = screen.getByRole('button', { name: 'Open Menu'})
```

Comment:
‘Open Menu’ here should be ‘Change Language’ or ‘Choose Language’ or similar.

Reply:
On zooniverse.org the label is the word ‘Language’, translated for the current locale.

Reply:
I've changed that label to 'Change Language', in the English `components.json` file.